kani-compiler/src/codegen_cprover_gotoc/codegen/operand.rs

```diff
@@ -439,7 +439,7 @@ impl<'tcx> GotocCtx<'tcx> {
 
     fn codegen_allocation_data(&mut self, alloc: &'tcx Allocation) -> Vec<AllocData<'tcx>> {
         let mut alloc_vals = Vec::with_capacity(alloc.relocations().len() + 1);
-        let pointer_size = self.symbol_table.machine_model().pointer_width() as usize / 8;
+        let pointer_size = self.symbol_table.machine_model().pointer_width as usize / 8;
```

Comment:
Looks like this isn't used anywhere else, but let's actually add an impl back to MachineModel for `pointer_width_in_bytes` instead of doing that here?

Reply:
doing that!